lib/active_merchant/billing/gateways/vantiv.rb

```diff
@@ -10,7 +10,7 @@ module Billing #:nodoc:
     # name.
     class VantivGateway < Gateway
       self.test_url = "https://www.testlitle.com/sandbox/communicator/online"
```

Comment:
Is there an updated test url?

Reply:
There was not at that time, but there is [now](https://developer.vantiv.com/message/3721). Updating.